doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,73 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple, module responsible for aggregating sensor data in form of sensor_event and passing them further in packages.
+
+Configuration
+*************
+
+The following Kconfig options are required:
+
+* :kconfig:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` - This option enables the |sensor_data_aggregator|.
+
+To use the module, you must complete the following steps:
+
+1. Enable the :kconfig:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` option.
+#. Add the app.overlay file that describes the aggregator parameters.
```

Comment:
I wouldn't say you need to add `app.overlay` specifically as it could be added in many places that would do the same.
Lets write:
`Enable aggregator in devicetree file`. Line 23 could be removed imho.

Reply:
I will leave decision if we should leave line: "   Each aggregator should be placed as a separate node."
to @divipillai.